tests/Export-DbaLogin.Tests.ps1

```diff
@@ -17,48 +17,45 @@ Describe "$CommandName Integration Tests" -Tags "IntegrationTests" {
     BeforeAll {
         $DefaultExportPath = Get-DbatoolsConfigValue -FullName path.dbatoolsexport
         $AltExportPath = "$env:USERPROFILE\Documents"
-        try {
```

Comment:
Removed `try` from BeforeAll and AfterAll since it blocks us from knowing when the initial setup is flawed (spoiler alert - it was).